defaults/main.yml

```diff
@@ -4,7 +4,7 @@ zookeeper_url: http://www.us.apache.org/dist/zookeeper/zookeeper-{{zookeeper_ver
 
 # Flag that selects if systemd or upstart will be used for the init service:
 # Note: by default Ubuntu 15.04 and later use systemd (but support switch to upstart)
-zookeeper_debian_systemd_enabled: "{{ ansible_distribution_version|version_compare(15.04, '>=') }}"
+zookeeper_debian_systemd_enabled: "{{ ansible_distribution_version|version_compare(15.04, '>=') or ansible_distribution_version|version_compare(8.0, '>=') }}"
```

Comment:
now this check includes all versions from `8.0` please change or condition to or (debian and version >= 8.0)